src/lib/Microsoft.Fx.Portability.Azure/Storage/AzureStorageExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.WindowsAzure.Storage.Blob;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Fx.Portability.Azure.Storage
+{
+    /// <summary>
+    /// Provides useful extensions to Azure Storage classes
+    /// </summary>
+    public static class AzureStorageExtensions
+    {
+        public static async Task<IEnumerable<CloudBlobContainer>> ListContainersAsync(this CloudBlobClient cloudBlobClient, string prefix = null)
+        {
+            BlobContinuationToken continuationToken = null;
+            var results = new List<CloudBlobContainer>();
+            do
```

Comment:
nit: new line

There are some other ones in the files down there but I didn't want to overwhelm your inbox

Reply:
Fixed

---
In reply to: [178096705](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178096705) [](ancestors = 178096705)